2021-09-15 22:52:11,176 INFO    MainThread:3341 [wandb_setup.py:_flush():69] setting env: {}
2021-09-15 22:52:11,176 INFO    MainThread:3341 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-15 22:52:11,176 INFO    MainThread:3341 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-15 22:52:11,176 INFO    MainThread:3341 [wandb_setup.py:_flush():69] setting login settings: {'mode': 'offline'}
2021-09-15 22:52:11,176 INFO    MainThread:3341 [wandb_init.py:_log_setup():342] Logging user logs to /home/denis/AI/python/clasificador de imagen/kaggle brain tumor/src/wandb/offline-run-20210915_225211-2cruc30m/logs/debug.log
2021-09-15 22:52:11,176 INFO    MainThread:3341 [wandb_init.py:_log_setup():343] Logging internal logs to /home/denis/AI/python/clasificador de imagen/kaggle brain tumor/src/wandb/offline-run-20210915_225211-2cruc30m/logs/debug-internal.log
2021-09-15 22:52:11,177 INFO    MainThread:3341 [wandb_init.py:init():375] calling init triggers
2021-09-15 22:52:11,177 INFO    MainThread:3341 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-15 22:52:11,177 INFO    MainThread:3341 [wandb_init.py:init():395] re-initializing run, found existing run on stack: elr3t5rg
2021-09-15 22:52:11,178 INFO    MainThread:3341 [wandb_run.py:finish():1271] finishing run denisquant/kaggle_brain_tumor/elr3t5rg
2021-09-15 22:52:11,213 INFO    MainThread:3341 [jupyter.py:save_history():353] not saving jupyter history
2021-09-15 22:52:11,213 INFO    MainThread:3341 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-15 22:52:11,213 INFO    MainThread:3341 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-09-15 22:52:11,213 INFO    MainThread:3341 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-15 22:52:11,214 INFO    MainThread:3341 [wandb_run.py:_restore():1543] restore
2021-09-15 22:52:11,666 INFO    MainThread:3341 [wandb_run.py:_wait_for_finish():1693] got exit ret: 
2021-09-15 22:52:12,157 INFO    MainThread:3341 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}

2021-09-15 22:52:13,229 INFO    MainThread:3341 [wandb_init.py:init():424] starting backend
2021-09-15 22:52:13,229 INFO    MainThread:3341 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-15 22:52:13,231 INFO    MainThread:3341 [backend.py:ensure_launched():135] starting backend process...
2021-09-15 22:52:13,239 INFO    MainThread:3341 [backend.py:ensure_launched():139] started backend process with pid: 3467
2021-09-15 22:52:13,240 INFO    MainThread:3341 [wandb_init.py:init():429] backend started and connected
2021-09-15 22:52:13,252 INFO    MainThread:3341 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-09-15 22:52:13,252 INFO    MainThread:3341 [wandb_run.py:_label_probe_notebook():822] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-15 22:52:13,252 INFO    MainThread:3341 [wandb_init.py:init():477] updated telemetry
2021-09-15 22:52:13,253 INFO    MainThread:3341 [wandb_init.py:init():540] starting run threads in backend
2021-09-15 22:52:15,532 INFO    MainThread:3341 [wandb_run.py:_console_start():1601] atexit reg
2021-09-15 22:52:15,532 INFO    MainThread:3341 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-15 22:52:15,532 INFO    MainThread:3341 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-15 22:52:15,533 INFO    MainThread:3341 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-15 22:52:15,533 INFO    MainThread:3341 [wandb_init.py:init():565] run started, returning control to user process
